src/components/TextInput/BaseTextInput/index.js

```diff
@@ -262,7 +262,7 @@ function BaseTextInput(props) {
                     style={[
                         props.autoGrowHeight && styles.autoGrowHeightInputContainer(textInputHeight, variables.componentSizeLarge, maxHeight),
                         !isMultiline && styles.componentHeightLarge,
-                        ...props.containerStyles,
+                        props.testInputWrapper,
```

Comment:
Why do we not need to spread this prop and why is this prop not in the propTypes? This PR is a bit confusing, could you add a bit more context? @perunt @allroundexperts Thanks!

Reply:
Good point, I've updated it to match our style guide. Thanks!